app/javascript/components/app.jsx

```diff
@@ -71,10 +72,19 @@ const client = new ApolloClient({
   },
 });
 
+const WidgetWindow = () => {
+  return <WidgetController client={client} />;
+};
+
 const App = () => (
   <ApolloProvider client={client}>
     <GlobalStyle />
-    <WidgetController client={client} />
+    <Router>
+      <Switch>
+        <Route path="/" component={WidgetWindow} />
```

Comment:
I think we can omit this path, no? A missing city name isn't particularly useful to us since the plan is to do away with `PRIMARY_LOCATION` entirely.

Reply:
Should the "/" path return a 404 error then, and only URLs with a city attached to the slug are valid?

Reply:
It depends (TM).

I think the rails router should redirect `/` to `"/#{AppEnv['PRIMARY_LOCATION']}"`. This is to avoid `/` on the client side

I think the client side should skip handling `/` for now. Now the client side is agnostic of PRIMARY_LOCATION.

I'll add new tickets for:
* Client side route `/` should display some kind of prompt to select a location.
* Remove the server side redirect for `/`
* Remove PRIMARY_LOCATION
* :tada: the death of PRIMARY_LOCATION